src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final int MAX_POSSIBLE_NUMBER = 101;
```

Comment:
naming is a bit confusing now because it says that the max possible number is 101, when in fact max number is 100, we should think about how we can make it more clear and readable